sql/store_repository.py

```diff
@@ -0,0 +1,19 @@
+from base_repository import BaseRepository
+
+
+class StoreRepository(BaseRepository):
+    def __init__(self):
+        super().__init__()
+
+    def create_table(self, table_name, column_1, type_1, column_2, type_2, column_3, type_3):
+        self.cursor.execute(f"create table {table_name} ({column_1} {type_1},{column_2} {type_2},{column_3} {type_3});")
+
+    def create_table_with_condition(self, table_name, column_1, type_1, column_2, type_2, column_3, type_3, condition):
```

Comment:
duplication of the code. You can make condition None bu default, and if it is not none add it to query in previous function